gradio/external.py

```diff
@@ -616,29 +761,32 @@ def load_chat(
     start_message = (
         [{"role": "system", "content": system_message}] if system_message else []
     )
+    file_types = [file_types] if isinstance(file_types, str) else file_types
```

Comment:
Suggest raising an error if an invalid file type is included. For example, in my testing I accidentally typed `images` instead of `image`

Reply:
added validation further down